server.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const express = require('express');
+
+// Constants
+const PORT = 3000;
+
+// App
+const app = express();
+
+app.use(express.static('public'));
+
+app.get('/', function (req, res) {
+   res.sendFile(__dirname + "/" + "index.html");
```

Comment:
Render is for when you have some server side templating to do. In this case, everything is rendered on the client via angularjs so sendfile is actually fine. Some nit-picky stuff: 

We always use either single quotes `'foo'` or back ticks

In this case, since we're concatenating strings, back ticks are appropriate as it gives us string templates:
```diff
- res.sendFile(__dirname + "/" + "index.html");
+ res.sendFile(`${__dirname}/index.html`);
```